helpers/generate_root_module.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+import sys
+
+
+OUTPUT_OBJECT_PATTERN = re.compile(r"""
```

Comment:
@aaron-lane one of the things I'd like reviewed or discussed is this regular expression. Since I'm trying to parse a context free grammar with a regular expression this is quite brittle, and I think we'd be better served by using a Python HCL parser to handle this - but that'll mean pulling in an outside dependency which is also problematic. What's your take on this approach?

Reply:
I think using a package like [pyhcl](https://github.com/virtuald/pyhcl) is preferable to a complex regular expression. pyhcl in particular only depends on [ply](https://github.com/dabeaz/ply/tree/master) which appears to have no dependencies of its own. Does that satisfy the concern about external dependencies?

Reply:
Yeah, anything is better than this regular expression. We can throw a `requirements.txt` in either `.` or `./helpers` to handle the dependency definition, do you have a preference?

Reply:
I think we should add a second directory under `./helpers` to isolate this particular script and its requirements from other scripts.